docs/guides/development/cli.md

```diff
@@ -21,14 +21,14 @@ The easiest way to get started it to follow the tutorial [Get started on your lo
 [View Tutorial][quickstart_cli]{ .md-button }
 
 
-### Requirements
+### Requirements {: #requirements}
```

Comment:
What does this syntax mean (`{: #requirements}`)? When I preview the markdown on github it renders that as normal text

Reply:
This sets an ID on the heading element, for consistent linking. It uses the Attribute List extension to Python markdown:

https://python-markdown.github.io/extensions/attr_list/

You can previous the fully rendered markdown using the link provided in the Read the Docs check below. For this PR, that's:

https://coda-packs-sdk--1843.com.readthedocs.build/en/1843/guides/development/cli/#migrating-from-the-web-editor